src/index.js

```diff
@@ -0,0 +1,185 @@
+  pressure: document.querySelector('.js--pressure'),
+  humidity: document.querySelector('.js--humidity'),
+  wind: document.querySelector('.js--wind'),
+  icon: document.querySelector('.js--icon'),
+  dateAndTime: document.querySelector('.date-box') 
+}
+
+const handleErrorResponse = (error) => {
+  console.error(error)
+}
+
+const registerListeners = () => {
+  searchForm.addEventListener('submit', onSearchSubmit)
+  displayDateAndTime();
+}
+
+const onSearchSubmit = async (event) => {
+  event.preventDefault()
+  
+  const data = new FormData(event.target)
+  const city = data.get('city')
+
+  await fetchWeatherData(city)
+  await fetchNextWeekWeatherData(city)
+}
+
+const fetchWeatherData = async (city) => {
+  try {
+    const request = await fetch(`${API_URL}&q=${city}&APPID=${API_KEY}`)
+    const response = await request.json()
+
+    if (response.cod !== 200) {
+      handleErrorResponse()
+    } else {
+      API_COUNTRY_CODE = response.sys["country"].toLowerCase();
+      displayWeather(response)
+    }
+
+  } catch (exception) {
+    // TODO: Handle fetch error (ex. Cannot connect to the remote host)
+  }
+}
+
+const displayWeather = function(data) {
+  DOM.cityName.innerHTML = data.name
+  DOM.weatherDesc.innerHTML = data.weather[0].description
+  DOM.temp.innerHTML = Math.round(data.main.temp) + ` ${UNITS.temperature}` 
+  DOM.tempMin.innerHTML = Math.round(data.main.temp_min) + ` ${UNITS.temperature}` 
+  DOM.tempMax.innerHTML = Math.round(data.main.temp_max) + ` ${UNITS.temperature}` 
+  DOM.pressure.innerHTML = data.main.pressure + ` ${UNITS.pressure}` 
+  DOM.humidity.innerHTML = Math.round(data.main.humidity) + ` ${UNITS.humidity}` 
+  DOM.wind.innerHTML = data.wind.speed + ` ${UNITS.wind}` 
+  DOM.icon.src = getWeatherImageURL(data.weather[0].icon)
+}
+
+const getWeatherImageURL = (iconID) => {
+  return `${API_IMG_URL}${iconID}@2x.png`
+}
+
+const fetchNextWeekWeatherData = async (city) => {
+  try {
+    const requestUrl = `${API_NEXTWEEK_URL}&q=${city},${API_COUNTRY_CODE}&APPID=${API_KEY}` // instead of pl make it fetchCountryCode, but how???
+    const request = await fetch(requestUrl);
+    const response = await request.json()
+
+    if (response.cod != 200) {
+      handleErrorResponse()
+    } else {
+      displayNextWeekWeather(response)
+    }
+    
+  } catch (exception) {
+      // TODO: Handle fetch error (ex. Cannot connect to the remote host)
+  }
+}
+
+const displayNextWeekWeather = (fetchedData, countryCode) => {
+
+  let nextDays = document.querySelector('.next-days');
+  // listOfStates is a list with 3 hour updates
+  let listOfStates = fetchedData.list;
+  // for the next 5 days, I will take only some data from this list mostly, by some data I mean data with interval = step * (i + 1), it could be e.g. 21 hours interval 
+  let step = Math.floor(listOfStates.length / 5) - 1; 
+  
+  nextDays.innerHTML = ""; // empty the box for nextWeekWeather
+  let tomorrow = new Date();
```

Comment:
Tutaj  bym pomyślał nad inną nazwą zmiennej. To nie jest `tomorrow` a zaincjalizowany obiekt daty :) 
Może
```
let date = new Date();
```

Chyba że chcemy mieć faktycznie `tomorrow` to może
```
let tomorrow = new Date().getDate() + 1;
```